Aakanksha/security w (#215)
NuevoFoundation/workshops: content/english/security-phishing/3-A closer look/_index.md

---
title: "A closer look"
description: ""
draft: false
weight: 3

> translationKey: "security-3"

Reviewer: ```suggestion
```

---

> #### Bob received the following email the other night.

Reviewer: ```suggestion
Bob received the following email the other night.
```

![Cannot load image](email.png?classes=border,shadow)

> #### What do you think is suspicious here? Let's unpack

Reviewer: ```suggestion
What do you think is suspicious here? Let's analyze the email:
```

<div style="text-align: center;">
<p style="color:purple">
<em>The sender's email address is suspicious and you don't know this email sender personally.</em></p>
</div>

![Cannot load image](subject.JPG?classes=border,shadow)

<div style="text-align: center;">
<p style="color:purple">
<em>Lucrative offer in the subject line suggests it's too good to be true.</em></p>
</div>

![Cannot load image](date.JPG?classes=border,shadow)

 <div style="text-align: center;">
<p style="color:purple">
<em> You received an email during irregular business hours.</em></p>
</div>

![Cannot load image](urgency.JPG?classes=border,shadow)

<div style="text-align: center;">
<p style="color:purple">
<em>These words show a sense of urgency.</em></p>
</div>

![Cannot load image](url.JPG?classes=border,shadow)

<div style="text-align: center;">
<p style="color:purple">
<em>The sender is asking you to click on a link or open an attachment that you don't know about.
The URL or the link has a spelling error which is hard to notice.</em></p>
</div>
<br>
After explaining all of this, Alice asked Bob to change all his passwords and bank account login credentials to be safe. 

> Thankfully, the attacker <b>Eve</b> had not yet acted on the information given by Bob and he was safe. 

Reviewer: ```suggestion
Thankfully, the attacker **Eve** had not yet acted on the information given by Bob and he was safe. 
```
